src/operator/subgraph/tensorrt/onnx_to_tensorrt.cc

```diff
@@ -127,16 +128,15 @@ nvinfer1::ICudaEngine* onnxToTrtCtx(
       }
       throw dmlc::Error("Cannot parse ONNX into TensorRT Engine");
   }
-
-  bool fp16 = trt_builder->platformHasFastFp16();
-
+  if (dmlc::GetEnv("MXNET_TENSORRT_USE_FP16", true)) {
+    if (trt_builder->platformHasFastFp16()) {
+      trt_builder->setFp16Mode(true);
+    } else {
+      LOG(INFO) << "WARNING: TensorRT can't use fp16 on this plateform";
```

Comment:
plateform -> platform

Reply:
Also we're logging INFO level logs but have WARNING in the message.  I'd remove the warning from the message and set log level to warning.  (this is a common issue in our codebase).